tests/jquery.flot.composeImages.Test.js

```diff
@@ -0,0 +1,674 @@
+/* eslint-disable */
+/* global $, describe, it, xit, xdescribe, after, afterEach, expect*/
+
+describe("composeImages", function() {
+    var placeholder, plot;
+    var composeImages = $.plot.composeImages;
+
+
+    function matchPixelColor(pixelData, r, g, b, a) {
```

Comment:
Please move these helper functions at the bottom of the file.

Reply:
done